### DISCUSSION TITLE
Howto goteleport+switch

### DISCUSSION THREAD

**Discussion by DyO63** [gravitational/teleport#12294] [Q&A]
I'm just discovering goteleport and i was wondering if it could be possible to use it  for manage all my ssh connections switchs ?
I see that it wasn't necessary to setup goteleport on a node to access it.

Is it true ?
Would it be possible to use it with my switchs ?

Thanks

**Comment by Valien** [comment]
Hey @DyO63 ! We do not support switches from a native integration standpoint (at this time) but you can easily leverage Teleport to manage access to certain VM's that do have access to switches, etc. We also do integrate with OpenSSH fairly well and that can be an option as well.

**Reply by webvictim** [reply]
https://goteleport.com/docs/server-access/guides/openssh/#configure-an-openssh-server-to-join-a-teleport-cluster
